packages/@sanity/form-builder/src/inputs/ArrayInput/typedefs.ts

```diff
@@ -1,12 +1,14 @@
 import {Type} from '../../typedefs'
 
+export type ValidEditModalValues = 'modal' | 'fullscreen' | string
+
 export type ArrayType = Type & {
   name: string
   title: string
   description: string
   readOnly: boolean | null
   options: {
-    editModal?: 'fold' | 'modal' | string
+    editModal?: ValidEditModalValues
```

Comment:
Couldn't this type just be named `ModalType` ?

Reply:
Yes, I agree. Will rename!